package.json

```diff
@@ -30,5 +30,9 @@
     "predeploy": "npm run build",
     "deploy": "gh-pages -d build",
     "prepublish": "npm run build"
+  },
+  "dependencies": {
```

Comment:
Why not just add it as a peer/dev dependency and then the demo will still build locally webpack doesn't care if something is a dev or real dependency. 

Then when you npm install react-slot-fill it won't try to bring React with it...

I agree in the long run with moving the demos but in the short run I think it's no big deal.